api/models/route.go

```diff
@@ -95,7 +95,7 @@ func (r *Route) Validate() error {
 		return ErrRoutesInvalidType
 	}
 
-	if r.Format != FormatDefault && r.Format != FormatHTTP {
+	if r.Format != FormatDefault && r.Format != FormatHTTP && r.Format != FormatJSON {
```

Comment:
at least note to self (not blocking), we could add `[]possibleFormats` to declarations in models so this falls out in the future